packages/repository/src/legacy-juggler-bridge.ts

```diff
@@ -160,4 +201,12 @@ export class DefaultCrudRepository<T extends Entity, ID>
   exists(id: ID, options?: Options): Promise<boolean> {
     return ensurePromise(this.modelClass.exists(id, options));
   }
+
+  protected toEntity(model: DataObject<T>): T {
+    return new this.entityClass(model.toObject()) as T;
+  }
+
+  protected toEntities(models: AnyObject): T[] {
```

Comment:
Should `models` be `Array<DataObject<T>>`?

Reply:
It should, but I was not able to find a way how to tell TypeScript compiler that when I invoke the `create` function with an array, then it will return back an array too. I'll take another look.

Reply:
Done in 878ec6b6